parquet/src/file/metadata.rs

```diff
@@ -462,6 +466,46 @@ impl ColumnChunkMetaData {
         self.statistics.as_ref()
     }
 
+    /// Returns `true` if this column chunk contains an offset index offset, `false` otherwise.
+    pub fn has_offset_index_offset(&self) -> bool {
+        self.offset_index_offset.is_some()
+    }
+
+    /// Returns the offset for the offset index.
+    pub fn offset_index_offset(&self) -> Option<i64> {
+        self.offset_index_offset
+    }
+
+    /// Returns `true` if this column chunk contains a offset index offset, `false` otherwise.
+    pub fn has_offset_index_length(&self) -> bool {
```

Comment:
Not sure all 4 'has_' functions are required, maybe a single 'has_column_index' will suffice